tests/stub/routing.py

```diff
@@ -56,33 +60,67 @@ def tearDown(self):
         self._writeServer3.reset()
         super().tearDown()
 
-    def router_script(self):
+    def router_script_adb(self):
         return """
         !: BOLT #VERSION#
         !: AUTO RESET
         !: AUTO GOODBYE
 
         C: HELLO {"scheme": "basic", "credentials": "c", "principal": "p", "user_agent": "007", "routing": #HELLO_ROUTINGCTX# #EXTRA_HELLO_PROPS#}
         S: SUCCESS {"server": "#SERVER_AGENT#", "connection_id": "bolt-123456789"}
-        C: ROUTE #ROUTINGCTX# [] "adb"
+        C: ROUTE #ROUTINGCTX# "*" "adb"
```

Comment:
Was there some reason to make it more lenient?

Reply:
Yes. This is the "default" router script that is being used in most tests. Some include bookmarks, others don't.

Reply:
This script is used in different test scenarios which are not testing bookmarks, maybe It's because of this.

Reply:
Indeed. Additionally, there are dedicated tests for checking if the driver sends bookmarks when fetching the RT.

Reply:
While I would not use it for testing bookmarks, I do like when scripts match my expectation as close as possible.